README.md

```diff
@@ -1,6 +1,6 @@
 
 # Project Name: Portfolio Skeleton Project
```

Comment:
- [ ] Please add a link to license file in your readme to make it easily accessible to other people :+1: